### PR TITLE
Webpack!

### PR DESCRIPTION
**Pull request by AWare**
This is all the handiwork of @tomverran 

New features: 
-Webpack instead of require
-NPM instead of bower (for js)
-Clean asset map and recompile on watch
-Future async loading to replace curl

### CODE REVIEW

**Comment by AWare** [comment]
Needs a new node install. 

Will be installing nvm

**Comment by AWare** [comment]
Teamcity server updated.

**Comment by paulbrown1982** [comment]
I get a JS error in comparisonTable.es6, the $ variable is an object not a function, $.default appears to be the function that does the selector.

**Comment by paulbrown1982** [comment]
Everything else works just fine, including contribute.

**Comment by jranks123** [comment]
screenshot at may 10 12-24-10

The columns of this table don't appear to be populated

File: frontend/assets/javascripts/src/modules/comparisonTable.es6 [hunk 1]
```diff
@@ -28,6 +28,7 @@ function tierSelector(tier) {
 // Instead they are split between rows, so we use data-tier to group them together.
 function addHoverListeners() {
     bean.on(COMPARISON_TABLE, 'mouseenter', CLICKABLE, e => {
+        console.log($);
```
Comment (paulbrown1982) [inline frontend/assets/javascripts/src/modules/comparisonTable.es6]: --

**Comment by AWare** [comment]
The build will fail if a non webpack build has been run on that agent, and will need this to be checked. 
image

**Comment by tomverran** [comment]
👍

**Pull request merged**